### PR TITLE
added waiting for silence in receiver

### PR DESCRIPTION
**Pull request by PawelA**

### CODE REVIEW

File: libecho/examples/RawConnection/main.cc [hunk 1]
```diff
@@ -75,7 +75,7 @@ int main(int argc, char *argv[]) {
 
         printf("Receiving %d bytes. Low frequency: %d, high frequency: %d, window size: %d, mag limit: %d\n", bytes,
                freq1, freq2, winsize, magLim);
-        auto connection = EchoRawConnection::getBitEchoRawConnection(winsize, freq1, freq2, freq1, freq2, magLim);
+        auto connection = EchoRawConnection::getBitEchoRawConnection(winsize, freq1, freq2, freq1, freq2, 0, magLim);
```
Comment (Lorak-mmk) [inline libecho/examples/RawConnection/main.cc:78]: Czy tu ma zawsze być 0 czy jednak lepiej żeby było to ustawiane w parametrze?

File: libecho/include/EchoRawConnection.h [hunk 2]
```diff
@@ -29,7 +29,7 @@ class EchoRawConnection {
      * @return              Pointer to desired EchoRawConnection object.
```
Comment (Riyria-was-taken) [inline libecho/include/EchoRawConnection.h:30]: Please change documentation when changing method declaration

File: libecho/src/BitReceiver.cc [hunk 3]
```diff
@@ -228,3 +228,12 @@ int BitReceiver::receive(uint8_t *buffer, int size) {
 
     return size;
 }
+
+void BitReceiver::wait() {
```
Comment (Riyria-was-taken) [inline libecho/src/BitReceiver.cc]: Maybe rename it to something like skip? I think that wait doesn't explain enough well what this method does.

**Review by Riyria-was-taken — APPROVED** [review]

**Review by a-piasta — APPROVED** [review]

File: libecho/examples/RawConnection/main.cc (hunk 1 above)
Comment (PawelA) [reply libecho/examples/RawConnection/main.cc:78]: na razie 0

**Pull request merged**